App/Containers/TextileManager.js

```diff
@@ -32,6 +32,7 @@ class TextileManager extends React.PureComponent {
   }
 
   componentDidMount () {
+    this.props.appStateChange(this.props.currentAppState, AppState.currentState)
```

Comment:
Was thinking that triggering the first state transition from a UI file would guarantee that `AppState.currentState` had a value by now. Not the case. It's still `unknown`, but let's use that to detect this specific case of launching from terminated into foreground.